Tagged content items count

fmlharrison opened the pull request:
<img width="730" alt="screen shot 2018-02-02 at 16 50 13" src="https://user-images.githubusercontent.com/20514501/35744814-350fb706-0839-11e8-954b-ea785845ab69.png">


The count of tagged content for the topic page taxon and each listed child taxon.

https://govuk-topic-pages-protot-pr-10.herokuapp.com/

https://trello.com/c/BLTyclRn/45-add-a-count-of-tagged-content-items-to-the-list-of-child-taxons

fmlharrison committed:
Add count method on rummager search.

This returns the number of results returned from a
rummager search and is used to display the number of
tagged content for a taxon.

vanitabarrett reviewed (approved):
This looks good to me! 👍

fmlharrison committed:
Add tagged_content_count method to Taxon class

This method is used to return the count of tagged
content for a particular content_id.
